### PR TITLE
Update PyDonate

### PR DESCRIPTION
**Pull request by SuperQ**
* Pin to specific version.
* Update to Python 3.7.
* Add tag for easier deploys.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Review by kevinjos — APPROVED** [review]

File: .ansible-lint [hunk 1]
```diff
@@ -11,3 +11,5 @@ exclude_paths:
 - roles/nickjj.docker
 skip_list:
 - 701 # No 'galaxy_info' found, not needed.
+- 106 # Role name {} does not match ``^[a-z][a-z0-9_]+$`` pattern'                                                                                                                                                                                                                                                                                  │
+- 208 # File permissions not mentioned'                                                                                                                                                                                                                                                                                                             │
```
Comment (elimisteve) [inline .ansible-lint:15]: Trailing `'` on these 2 lines there for a reason?

Comment (elimisteve) [reply .ansible-lint:15]: There's also a `|` way to the right at the end of these 2 lines.

Comment (SuperQ) [reply .ansible-lint:15]: Whups, copy-pasta mistake.

File: .ansible-lint [hunk 2]
```diff
@@ -11,3 +11,5 @@ exclude_paths:
 - roles/nickjj.docker
 skip_list:
 - 701 # No 'galaxy_info' found, not needed.
+- 106 # Role name {} does not match ``^[a-z][a-z0-9_]+$`` pattern'
+- 208 # File permissions not mentioned'
```
Comment (elimisteve) [inline .ansible-lint:15]: @SuperQ There's still a trailing single quote on these two lines, FYI

**Pull request merged**